docs/guide/release-notes.asciidoc

```diff
@@ -14,6 +15,30 @@
 * <<rn-8-1-0>>
 * <<rn-8-0-0>>
 
+[discrete]
+[[rn-8-4-0]]
+=== 8.4.0 (2022-08-25)
+
+[discrete]
+==== Documents
```

Comment:
Can you move the knn parameter being added to search and async_search.submit under a heading "Search"